etcdserver/security/security_requests.go

```diff
@@ -111,6 +120,7 @@ func (s *Store) createResource(res string, value interface{}) (etcdserver.Respon
 	return s.setResource(res, value, false)
 }
 func (s *Store) setResource(res string, value interface{}, prevexist bool) (etcdserver.Response, error) {
+	s.ensureSecurityDirectories()
```

Comment:
can we add a cached result for this? if this is created, we do not have to send a raft command.